### ISSUE TITLE
Possibility to not use websockets?

### ISSUE THREAD

**Issue by seiyria** [node-inspector/node-inspector#513]
I have a setup where I have a server I'm trying to debug, which is on a private network, and behind a loadbalancer. Everything starts alright, and I've tried setting up an ssh tunnel to get into that particular server.

I set up an SSH tunnel like so:
```
ssh admin@my-server -L 8080:localhost:8080 -p 2200
```
which definitely gets me into my server correctly, however, I get this error:
```
WebSocket connection to 'ws://localhost:8080/ws?port=5858' failed: Error in connection establishment: net::ERR_EMPTY_RESPONSE
```

Any ideas on possible ways to get around this?

**Comment by bajtos** [comment]
What is the URL you entered into your browser?

**Comment by bajtos** [comment]
Am I assuming correctly that `http://localhost:8080/debug?port=5858` loads the node-inspector front-end in your browser, but then the websocket connection fails?

**Comment by seiyria** [comment]
What you said was correct, @bajtos - I was trying to access it over an SSH tunnel, and I think that was the URL. 

I actually eventually came to a way to get in without using an SSH tunnel, so I think I could close this issue now. I had to forward some ports on each level and I forgot to do it on one of them, so I was unsure of why the connection was failing, seemingly at random.

Sorry to open an issue over something silly!
